CMakeLists.txt

```diff
@@ -71,12 +71,10 @@ endif()
 
 if(NOT EVHTP_DISABLE_EVTHR)
     find_package(Threads REQUIRED)
-    if(TARGET Threads::Threads)
```

Comment:
> Having some build issues I think this might resolve

I don't think this change is related. Have you tried to use this patch? According to the link you provided Android toolchain is not working, however `Threads::Threads` target present on Android.

Reply:
I can try it now.

Reply:
Weirdly enough, Android still is not working, but I am getting better results. See here: https://travis-ci.org/isaachier/hunter/builds/359905667.

Reply:
May be because of `set(EVHTP_DISABLE_REGEX ON)`? Probably Threads::Threads part is not related at all?